tools/cert_create/Makefile

```diff
@@ -61,8 +54,7 @@ all: clean ${BINARY}
 
 ${BINARY}: ${OBJECTS} Makefile
 	@echo "  LD      $@"
-	@echo 'const char build_msg[] = "Built : "__TIME__", "__DATE__; \
-                const char platform_msg[] = "${PLAT}";' | \
```

Comment:
With platform_oid.h deprecated, the cert_create tool is no longer platform-dependent.
So, the message "Target platform ..." makes no sense.  What's why.

We should keep host tools platform-independent where possible because:

[1] We can avoid recompiling under tools/ directory for build-test across all platforms

[2] fiptool and cert_create may be useful for some people to manipulate FIP image outside the ATF project.
For example, developers working on U-Boot or EDK2 (=BL33) probably no interest in building ARM Trusted Firmware.   What they want to do is to generate a certificate for BL33 with cert_create and update the the fip.bin with fiptool, then flush the new fip.bin to the board.
So, they may finally decide to provide a package of the host tools for some distributions (such as deb, rpm package).  This will be pretty easy if the host tools have no dependency on a particular platform.

         $ sudo apt-get install atf-tools

Then, pre-built fiptool and cert_create will be installed in /usr/bin directory on their system.
Similar things often happen.  For example, Debian provides a package "u-boot-tools".

If the cert_create depends on a particular platform, this would be difficult.  If cert_create built for ARM Ltd. contains ARM's OID, it would not work for Socionext Inc.